app/models/group-model.coffee

```diff
@@ -44,6 +44,10 @@ global.cobudgetApp.factory 'GroupModel', (BaseModel) ->
       _.first _.filter @memberships(), (membership) ->
         membership.memberId == member.id
 
+    bucketsWaitingForPayment: ->
+      _.filter @buckets(), (bucket) ->
+        bucket.fundedAt && !bucket.paidAt
+
```

Comment:
this is already defined in `group-page-buckets.coffee`. presumably there is a place we could define this such that it would be available to both directives?